python/psautohint/ufoFont.py

```diff
@@ -1103,9 +1104,8 @@ def buildGlyphHashValue(self, width, outlineXML, glyphName,
                                          "yScale", "xOffset", "yOffset"]:
                         try:
                             value = childContour.attrib[transformTag]
-                            rval = eval(value)
-                            if int(rval) == rval:
-                                value = str(int(rval))
+                            if self._represents_integer(value):
```

Comment:
The `_represents_integer()` method seems like overkill. I think you can just do `value.isdigit()`.

I'm confused by the next line. If you know it's a string that represents an integer, why are you converting it to an integer and then back into a string?

Reply:
I considered `isdigit()`,  but it can't be used for validating an integer because it returns `False` for `+1`, `1.0`, `-1` (and perhaps others I haven't even thought of).

Reply:
OK... so do you still need the next line that converts a string to an integer back to a string?

Reply:
This is what I wrote in the commit notes,
> Not sure why integer-like string values need special handling here, but let's leave that for later.